File: Fisica/src/Fisica/Models/RespostaTopicoModel.cs
namespace Fisica.Models
{
    public class RespostaTopicoModel
    {
        public long Id { get; set; }
        public long TopicoForumId { get; set; }
        public string Descricao { get; set; }
        public string NomeUsuario { get; set; }
    }
}

File: Fisica/src/Fisica.Website/ClientApp/src/app/shared/models/resposta-topico.model.ts
import { Replica } from "./replica.model";

export class RespostaTopico {
  constructor(
    public id?: number,
    public topicoForumId?: number,
    public descricao?: string,
    public nomeUsuario?: string,
    public replicas?: Replica[]
  ) { }
}
